src/test/java/com/shapesecurity/shift/es2016/test262/Test262.java

```diff
@@ -0,0 +1,385 @@
+			"src/test/resources/test262/test/built-ins/AsyncFunction/AsyncFunction-name.js",
+			"src/test/resources/test262/test/built-ins/AsyncFunction/AsyncFunction-prototype.js",
+			"src/test/resources/test262/test/built-ins/AsyncFunction/instance-length.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-this-value-passed.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-default-that-throws.js",
+			"src/test/resources/test262/test/language/statements/async-function/syntax-declaration.js",
+			"src/test/resources/test262/test/language/statements/async-function/declaration-returns-promise.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-this-value-global.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-body.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-body-that-returns.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-body-that-throws.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-unmapped-arguments.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-body-that-returns-after-await.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-mapped-arguments.js",
+			"src/test/resources/test262/test/language/statements/async-function/evaluation-body-that-throws-after-await.js",
+			"src/test/resources/test262/test/language/statements/async-function/syntax-declaration-line-terminators-allowed.js",
+			"src/test/resources/test262/test/language/expressions/async-function/syntax-expression-is-PrimaryExpression.js",
+			"src/test/resources/test262/test/language/expressions/async-function/expression-returns-promise.js",
+			"src/test/resources/test262/test/language/expressions/await/await-awaits-thenable-not-callable.js",
+			"src/test/resources/test262/test/language/expressions/await/await-in-nested-function.js",
+			"src/test/resources/test262/test/language/expressions/await/await-awaits-thenables.js",
+			"src/test/resources/test262/test/language/expressions/await/await-BindingIdentifier-in-global.js",
+			"src/test/resources/test262/test/language/expressions/await/syntax-await-has-UnaryExpression.js",
+			"src/test/resources/test262/test/language/expressions/await/await-throws-rejections.js",
+			"src/test/resources/test262/test/language/expressions/await/await-in-nested-generator.js",
+			"src/test/resources/test262/test/language/expressions/await/await-awaits-thenables-that-throw.js",
+			"src/test/resources/test262/test/language/expressions/await/syntax-await-has-UnaryExpression-with-MultiplicativeExpression.js",
+			"src/test/resources/test262/test/language/expressions/async-arrow-function/arrow-returns-promise.js",
+
+			// yield not implemented
```

Comment:
This isn't true, these are bugs with our handling of `yield` in various contexts.